stable/artifactory-jcr/README.md

```diff
@@ -16,16 +16,19 @@ This chart will do the following:
 
 ## Installing the Chart
 
-### Add JFrog Helm repository
-Before installing JFrog helm charts, you need to add the [JFrog helm repository](https://charts.jfrog.io/) to your helm client
+### Add ChartCenter Helm repository
+
+Before installing JFrog helm charts, you need to add the [ChartCenter helm repository](https://chartcenter.io) to your helm client
+
 ```bash
-helm repo add jfrog https://charts.jfrog.io
+helm repo add center https://repo.chartcenter.io
+helm repo update
 ```
 
 ### Install Chart
 To install the chart with the release name `jfrog-container-registry`:
 ```bash
-helm install --name jfrog-container-registry jfrog/artifactory-jcr
+helm upgrade --install jfrog-container-registry --namespace jfrog-container-registry center/jfrog/artifactory-jcr  --version 1.4.1
```

Comment:
Why not helm install?

Reply:
This is  to support helm v2 and v3 . Same command can be used for install and upgrade . If release doesn't exist , its a fresh install :)

Reply:
chartcenter does not say this in  their install commands - why keep it  different? In this doc, we are saying it  is fresh  install anyway right? helm v3 does not  support install command?